src/HotChocolate/Core/src/Abstractions/IQuery.cs

```diff
@@ -0,0 +1,12 @@
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace HotChocolate
+{
+    public interface IQuery
```

Comment:
thoughts on IDataQuery instead of IQuery so there is no confusion betweeen graphql vs data...

Reply:
yup that's right. We renamed it to IExecutable